cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server_test.go

```diff
@@ -268,9 +272,15 @@ func makeServer(t *testing.T, authorized bool, actionConfig *action.Configuratio
 	}, mock, cleanup
 }
 
-func newServerWithSecretsAndRepos(t *testing.T, secrets []k8sruntime.Object, unstructuredObjs []k8sruntime.Object, repos []*v1alpha1.AppRepository) *Server {
+func newServerWithSecretsAndRepos(t *testing.T, secrets []k8sruntime.Object, repos []*v1alpha1.AppRepository) *Server {
```

Comment:
I needed something similar to this function, which was created for the updated repository tests, but noticed that every call-site was simply creating the unstructured objects from the repos that were also passed in. So I refactored so call-sites don't need to create and pass the unstructured objects at all.